.travis.yml

```diff
@@ -44,6 +44,16 @@ jobs:
       python: '3.7'
       sudo: required
       dist: xenial
+    - language: generic
```

Comment:
Awesome. 😎 

Is `python2.7` available on this image? If so, is it worth it to also test on 2.7?